pkg/generate/generate_controller.go

```diff
@@ -145,7 +146,7 @@ func (c *Controller) Run(workers int, stopCh <-chan struct{}) {
 		// Deletion of policy will be handled by cleanup controller
 	})
 
-	c.nsInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
+	c.namespaceInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
```

Comment:
Do we still need to add the event handler for namespaces? As a generate policy can be triggered by **any** resource, can we make it consistent to trigger policy reconciliation through webhook?

Reply:
Sure, will take a look into this.

Reply:
removed the handler